tsconfig.build.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  // Visit https://aka.ms/tsconfig to read more about this file
```

Comment:
`json` doesn't support comments like these.  Can you confirm that the `ts` is interpreting `tsconfig.build.json` and `tsconfig.json` as JS instead of JSON?